src/Place/Canonical/DBALDuplicatePlaceRepository.php

```diff
@@ -95,15 +97,39 @@ public function calculateNoLongerInCluster(): array
         return $this->processRawToClusterRecord($statement->fetchAllAssociative());
     }
 
-    public function addToDuplicatePlacesRemovedFromCluster(string $clusterId): void
+    /** @return ClusterRecord[] */
+    public function calculateNotYetInCluster(): array
```

Comment:
This method name is using an action that will happen later in its name. This is just a method to get the new clusters.